Model/Api/OAuthRedirect.php

```diff
@@ -365,11 +390,9 @@ public function login($code = '', $scope = '', $state = '', $reference = '')
                 $checkoutSession = $this->sessionHelper->getCheckoutSession();
                 if ($checkoutSession) {
                     $quote = $checkoutSession->getQuote();
-                    if ($quote) {
-                        $quote->setCustomer($customer);
-                        $quote->setCustomerIsGuest(false);
-                        $this->cartHelper->saveQuote($quote);
-
+                    if ($quote->getId()) {
+                        // we should merge guest customer cart with customer cart if login is not from bolt modal
```

Comment:
I guess `assignCustomer` is the function that merge two carts? If so, could we change the comment to "call the function that ..."

Reply:
comment was updated.